### PR TITLE
Ailith external review

### PR DESCRIPTION
**Pull request by ailithewing**
Minor changes again. 

Spaced out some equations, added some interpretation elements to an exercise.

Suggest mentioning that the size of the effect of the interaction term in one exercise is v small although significant. Worth highlighting how that may effect your interpretation.

Ailith

### CODE REVIEW

File: _episodes_rmd/04-assumptionsAndFit.Rmd [hunk 1]
```diff
@@ -156,7 +155,7 @@ without the interaction using the tools discussed in this episode.
 >
 > Extend this model by adding separate intercepts 
 > for the levels of the `Sex` variable. Visualise the model
-> using `interact_plot()`. 
+> using `interact_plot()`. Do you think that including the interaction between `Height` and `Sex` is necessary?
```
Comment (ezherman) [inline _episodes_rmd/04-assumptionsAndFit.Rmd:158]: The students are asked to add the `Sex` term without an interaction (i.e. separate intercepts). I like asking the question you proposed, but maybe we can phrase it as "Do you think that including separate intercepts across the levels of `Sex` is necessary?"

File: _episodes_rmd/04-assumptionsAndFit.Rmd [hunk 2]
```diff
@@ -168,7 +167,7 @@ without the interaction using the tools discussed in this episode.
 > >               plot.points = TRUE, interval = TRUE,
 > >               point.size = 0.7)
 > > ```
-> > 
+> > There is little evidence from the plot that the effect of height on weight is different between sexes meaning that an interaction term is not needed. But we could assess the model fit with and without the interaction to check.
```
Comment (ezherman) [inline _episodes_rmd/04-assumptionsAndFit.Rmd:170]: This would then also need to be changed to relate to separate intercepts rather than an interaction.

**Review by ezherman — CHANGES_REQUESTED** [review]

File: _episodes_rmd/04-assumptionsAndFit.Rmd (hunk 1 above)
Comment (ailithewing) [reply _episodes_rmd/04-assumptionsAndFit.Rmd:158]: Yep agreed that would be better.

File: _episodes_rmd/04-assumptionsAndFit.Rmd (hunk 2 above)
Comment (ailithewing) [reply _episodes_rmd/04-assumptionsAndFit.Rmd:170]: yep.

**Review by ezherman — COMMENTED** [review]
Good point regarding effect size vs significance - we don't really touch on this except for a small note in the simple linear regression lesson. 

I'm hesitant to include a note on statistical power, since this isn't a concept we discuss at all. And we teach the students to not pay attention to the p-values, so coming back to them in this episode may be confusing. We could include a note reminding students to look at the standard error though, which allows them to construct a 95% CI.

What plot are you referring to that suggests little effect of an interaction between age and sex? The interaction term of -0.02 (`Hemoglobin_Age_Sex`) actually changes the lines a fair bit in that model (see the interact_plot). However the interaction term of -0.28 in the `BPSysAve_Age_Sex` model might not actually be necessary - I don't think that model is a great example of the need for an interaction. But at the time I didn't ifnd anything better.

File: _episodes_rmd/04-assumptionsAndFit.Rmd (hunk 1 above)
Comment (ezherman) [reply _episodes_rmd/04-assumptionsAndFit.Rmd:158]: Cool, I haven't found a way to make the change before merging, so I will make the change after approving the merge.

**Pull request merged**